waitress/app/utils.py

```diff
@@ -114,8 +134,8 @@ def normalize(info):
         except Exception as e:
             return "Users couldn't be updated successfully - %s" % e.message
 
-    @classmethod
-    def get_real_users(cls, difference, not_user):
+    @staticmethod
+    def get_real_users(difference, not_user):
```

Comment:
The `get_real_users` function receives the difference between the user list in the database and that from Slack. The `not user` argument is a list of bots, deleted users...
If a user was deleted at any time, this function helps remove them from the db, thus keeping the db updated.